lib/compressor/restructure/4-restructShorthand.js

```diff
@@ -118,6 +118,16 @@ TRBL.prototype.getValueSequence = function(value, count) {
             case 'Percentage':
                 break;
 
+            case 'Function':
+                switch (child.name) {
```

Comment:
Also looks like 

```js
special = child.name;
break;
```

is enough and improve optimisations for values with function